### ISSUE TITLE
Replace state title in BrowserProtocol is not available

### ISSUE THREAD

**Issue by christophediprima** [4Catalyzer/farce#61]
It is not possible to replace the title when using the FarceAction.replace because BrowserProtocol always set it to null.

https://github.com/4Catalyzer/farce/blob/d3e6e0cdd6a6ee43b7acc8d8b67ed00cefc76ce1/src/BrowserProtocol.js#L66

My use case is following:

1) I am on listing page and my page title is : "(5) unseen items"
2) I click on an unseen item and navigate to the detail page, my title change to : "(4) unseen items"
3) I close the detail page and execute a FarceAction.go(-1)
4) My browser still displays "(5) unseen items" even if my head title has changed.

**Comment by taion** [comment]
The `title` argument to `pushState` and `replaceState` doesn't generally do anything, at least on either Chrome or Firefox, and correspondingly this library doesn't do anything to control those titles, at least as far as I'm aware.

Aren't you using something like React Helmet to set the title?

**Comment by christophediprima** [comment]
You are right I noticed the the tilte param is ignored. I am using Helmet, but the browser doesn't read the title when you change the page title then do an history back... the browser will display the previous title. It is ok if you first do history back then change the title. I fixed my issue by using a different title for my detail and list page.
